libs/components/indicators/src/lib/modules/alert/alert.component.spec.ts

```diff
@@ -50,13 +50,14 @@ describe('Alert component', () => {
     cmp.closeable = true;
     fixture.detectChanges();
 
-    const attributes: any = el.querySelector('.sky-alert-close').attributes;
-    expect(attributes['hidden']).toBe(undefined);
+    const attributes: NamedNodeMap | undefined =
```

Comment:
For strict mode, I had to give `attributes` a proper type, which then forced me to replace the `['hidden']` syntax with `NamedNodeMap`'s `getNamedItem()` function, which returns `null`, instead of `undefined`.